documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,141 @@
+   11. The name cannot contain any trademarks (such as "Nike") unless the organization team is formally authorized, in writing, to use such names.
+   12. Names should not contain any unnecessary parts in the name, like "competition".
+   13. The competition nickname and ID must resemble the competition name.
+       1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters). Exception: groups of competitions may have modified nicknames and IDs to ensure consistency with past and future competitions.
+2. Location(s)
+   1. A location consists of the country, city, and venue.
+   2. The location should also include subregions of the country that are common to include with addresses (e.g. the state a U.S. city is located in).
+   3. Competitions will be accepted if there are no other competitions within 100 km driving distance and less than 19 days away.
+      1. Otherwise, the WCA Board should review the circumstances of the proposed competition. A Delegate requesting approval for such a competition must provide further information on why the competition should still be accepted. Possible arguments that support such a competition being accepted include:
+         1. The two competitions don’t have events in common.
+         2. The two competitions take place in different countries.
+   4. 3x3x3 Fewest Moves simultaneous competitions
+      1. 3x3x3 Fewest Moves simultaneous competitions are WCA competitions having only the 3x3x3 Fewest Moves event and taking place simultaneously in different locations.
+         1. If any location of such competitions is closer than 100 km to any other competition, then
+            1. both competitions should be at least 5 days apart, or
+            2. the other competition must not have the 3x3x3 Fewest Moves event.
+      2. A 3x3x3 Fewest Moves simultaneous competition cannot be part of any other regular competition.
+         1. Time frames when both are held cannot overlap.
+   5. Twin competitions
+      1. Twin competitions are WCA Competitions that are held less than 5 days apart for which the distance between them is less than 100 km, with the intention to accommodate more competitors to compete in one area.
+         1. Twin competitions may be accepted if the need for competitions in an area is so great that the community will benefit from having multiple competitions held in a short period of time.
+      2. Twin competitions are obliged to clearly and publicly state their condition of twin competitions and their special participation terms.
+         1. A competitor is only permitted to participate in one of the twin competitions.
+            1. If a competitor competes at more than one of the involved twin competitions, only the results from the competition that they compete in first will be kept.
```

Comment:
Has this ever been used? I think it's fine keeping it for someone to utilize in the future, just want to clarify that that hasn't happened yet.

I'd want to add the restriction that a competitor may not **register** for both competitions.

Reply:
It was *almost* used last year (Sam and I almost both delegated competitions in NYC on the same date). I think there may be cases that exist before I joined the Board.

I think it's reasonable that they can only register for one (but I think it's also reasonable that they can change their registration if a second competition is announced when they're already registered for the first). In an ideal world, our software handles this all cleanly, but it's a lot of work for something that is almost never used.

Reply:
Perhaps add a condition that the twin competitions be announced jointly, or that registration for the first does not open until the second is announced? I feel that would make it fairer and remove the difficulty of making it very simple to switch registrations. 

I could see this become relevant in areas where the effort of organising local competitions for 200+ competitors becomes too much - it could be cheaper and less work to organise two sets of 100 person competitions.

Reply:
Specific suggested changes are appreciated if someone feels strongly about changing this.

Reply:
I'm resolving this conversation now, let's discuss this for version 1.1 :stuck_out_tongue: